### ISSUE TITLE
Switch to new checkpoint

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#106]
```
WARNING:tensorflow:*******************************************************
WARNING:tensorflow:TensorFlow's V1 checkpoint format is deprecated; V2 will become the default shortly after 10/31/2016.
WARNING:tensorflow:Consider switching to the more efficient V2 format now:
WARNING:tensorflow:   `tf.train.Saver(write_version=tf.train.SaverDef.V2)`
WARNING:tensorflow:to prevent breakage.
WARNING:tensorflow:*******************************************************
```

**Comment by lissyx** [comment]
https://github.com/tensorflow/tensorflow/commit/0f93134fc7deac6ddf3d443ab382cd06d37543af#diff-66cdb2259545b655a2491a0b05185a23

**Comment by kdavis-mozilla** [comment]
The fix is in the issue106 branch, but we should't merge it as it requires a new 0.11rc version of TF

**Comment by Cwiiis** [comment]
Could we put this in a try/catch and apply it so we work in both old and new versions?

**Comment by kdavis-mozilla** [comment]
Fixed with @lissyx TF 1.0.0 commit
